README.md

```diff
@@ -9,7 +9,7 @@ NodeJS HTTP client based on [Request-Promise](https://github.com/request/request
 
 Once you build a canvas instance, you get an object with only four methods:
 
-1. The low-level `requestUrl()` method to perform any request with any HTTP method
+1. The low-level `requestUrl()` method to perform a request with any HTTP method
 2. `get()` to perform a GET request
```

Comment:
Was thinking about adding some comments for `get` and the `list`/`listPaginated` below, noting how to supply a query string - whaddya think?

Reply:
Added some new text to the README!